src/main/java/seedu/duke/ui/Formatter.java

```diff
@@ -47,10 +47,92 @@
+            + LS
+            + "`------'   `--'       `------' `$$$$$$$$$$$$$$$$$$$$$$$F:??$$$:)$$$$P\",. $$F"
+            + LS
+            + "                                ?$$$$$$$$$$$$$$$$$$$$$$(   `$$ J$$F\"d$$F,$F"
+            + LS
+            + "                                 ?$$$$$$$$$$$$$$$$$$$$$h,  :P'J$$F  ,$F,$"
+            + LS
+            + "                                  ?$$$$$$$$$$$$$$$$$$$$$$$ccd$$`$h, \",d$"
+            + LS
+            + "                                   \"$$$$$$$$$$$$$$$$$$$$$$$$\",cdc $$$$\""
+            + LS
+            + "                           ,uu,      `?$$$$$$$$$$$$$$$$$$$$$$$$$$$c$$$$h"
+            + LS
+            + "                     .,d$$$$$$$cc,   `$$$$$$$$$$$$$$$$??$$$$$$$$$$$$$$$,"
+            + LS
+            + "                   ,d$$$$$$$$$$$$$$$bcccc,,??$$$$$$ccf `\"??$$$$??$$$$$$$"
+            + LS
+            + "                  d$$$$$$$$$$$$$$$$$$$$$$$$$h`?$$$$$$h`:...  d$$$$$$$$P"
+            + LS
+            + "                 d$$$$$$$$$$$$$$$$$$$$$$$$$$$$`$$$$$$$hc,,cd$$$$$$$$P\""
+            + LS
+            + "             =$$?$$$$$$$$P' ?$$$$$$$$$$$$$$$$$;$$$$$$$$$???????\",,"
+            + LS
+            + "                =$$$$$$F       `\"?????$$$$$$$$$$$$$$$$$$$$$$$$$$$$$bc"
+            + LS
+            + "                d$$F\"?$$k ,ccc$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$i"
+            + LS
+            + "        .     ,ccc$$c`\"\"u$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$P\",$$$$$$$$$$$$h"
+            + LS
+            + "     ,d$$$L  J$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$\" `\"\"$$$??$$$$$$$"
+            + LS
+            + "   ,d$$$$$$c,\"$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$F       `?J$$$$$$$'"
+            + LS
+            + "  ,$$$$$$$$$$h`$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$F           ?$$$$$$$P\"\"=,"
+            + LS
+            + " ,$$$F?$$$$$$$ $$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$F              3$$$$II\"?$h,"
+            + LS
+            + " $$$$$`$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$P\"               ;$$$??$$$,\"?\""
+            + LS
+            + " $$$$F ?$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$P\",z'                3$$h   ?$F"
+            + LS
+            + "        `?$$$$$$$$$$$$$$$??$$$$$$$$$PF\"',d$P\"                  \"?$F"
+            + LS
+            + "            \"\"\"\"\"\"\"         ,z$$$$$$$$$$$$$P"
+            + LS
+            + "                                  J$$$$$$$$$$$$$$F"
+            + LS
+            + "                                 ,$$$$$$$$$$$$$$F"
+            + LS
+            + "                                 :$$$$$c?$$$$PF'"
+            + LS
+            + "                                 `$$$$$$$P"
+            + LS
+            + "                                  `?$$$$F";
+    // @@author
+
+
     public static String getNotusLogo() {
         return NOTUS_LOGO;
     }
 
+    public static String getGoodbyeMessage() {
+        return GOODBYE_MESSAGE;
+    }
+
```

Comment:
Want to move all the asci art and messages to another class?

Reply:
Good idea. Will do so!